node/src/components/consensus/era_supervisor/tests.rs

```diff
@@ -160,7 +210,7 @@ fn new_test_chainspec(stakes: Vec<(PublicKey, u64)>) -> Chainspec {
             GenesisAccount::new(pk.into(), pk.to_account_hash(), motes, motes)
         })
         .collect();
-    chainspec.genesis.timestamp = Timestamp::now() + 1000.into();
+    chainspec.genesis.timestamp = Timestamp::now();
```

Comment:
Why are we removing that? (I'm not sure why it was there in the first place, though :sweat_smile: )

Reply:
I had copied it from [the validator reactor test](https://github.com/CasperLabs/casper-node/blob/master/node/src/reactor/validator/tests.rs), where it's needed because it uses real time, and the genesis timestamp must be later than node initialization.

But in this test (so far, at least) this is a read-only node.